### PR TITLE
updating extensions nugets

### PR DESCRIPTION
**Pull request by brettsam**
This is specifically to pull in https://github.com/Azure/azure-webjobs-sdk-extensions/pull/433

### CODE REVIEW

File: src/WebJobs.Script.WebHost/Web.config [hunk 1]
```diff
@@ -242,6 +242,14 @@
         <assemblyIdentity name="System.Net.Http.Primitives" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
         <bindingRedirect oldVersion="0.0.0.0-4.2.29.0" newVersion="4.2.29.0" />
       </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity name="System.Diagnostics.Tracing" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-4.2.0.0" newVersion="4.2.0.0" />
+      </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity name="System.Security.Cryptography.Algorithms" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
```
Comment (fabiocav) [inline src/WebJobs.Script.WebHost/Web.config]: Need to make sure these redirects don't go in.

Comment (brettsam) [reply src/WebJobs.Script.WebHost/Web.config]: What are those showing up for? Any idea? I'll nuke them.

**Comment by brettsam** [comment]
Updated.

**Comment by brettsam** [comment]
Cosmos tests are failing -- I'll investigate in the morning.

**Comment by brettsam** [comment]
@ealsur -- it looks like we have a race in the Start/Stop calls now. I've filed issue https://github.com/Azure/azure-webjobs-sdk-extensions/issues/455. I'll keep this PR open and update it with a build when we have a fix.

**Comment by fabiocav** [comment]
Is this blocked on the issue you mentioned above?

**Comment by brettsam** [comment]
Yes it's blocked based on above. Talking with @ealsur about this.

**Pull request merged**